settings.gradle

```diff
@@ -16,3 +14,9 @@ gradleEnterprise {
         uploadInBackground = System.getenv("CI") == null
     }
 }
+
+rootProject.name = 'terra-cli'
+
+enableFeaturePreview('ONE_LOCKFILE_PER_PROJECT')
```

Comment:
nit blank line after this line. Actually, could this go somewhere higher in the file, since this is important?

Reply:
Sure, the line was auto-added.
Will move project name to the top